lambdas/tests/eventbrite/handlers/producer.test.js

```diff
@@ -165,7 +150,20 @@ describe("Eventbrite Producer", function() {
             pagination: { has_more_items: false, page_count: 3 }
           }
         );
-        done();
+        done(err);
+      });
+    });
+
+    it("returns file paths", function(done) {
+      producer.produce(event, context, function(err, response) {
+        expect(response).toEqual({
+          message: [
+            "path/to/new/file.json",
```

Comment:
NAB: this initially looked a bit odd to me as I expected the producer to dedupe the file paths but am I correct in saying this is a side effect of mocking ` uploadTo.mockImplementation(resolved({ key: "path/to/new/file.json" }));` and it shouldn't be an issue for producers?